FSJ_django20_project/FSJ/views_adjudicator.py

```diff
@@ -19,4 +22,100 @@ def is_adjudicator(usr):
 def adjudicator_home(request, FSJ_user):
     context = get_standard_context(FSJ_user)   
     template = loader.get_template("FSJ/home.html")
-    return HttpResponse(template.render(context, request))
\ No newline at end of file
+    return HttpResponse(template.render(context, request))
+
+@login_required
+@user_passes_test(is_adjudicator)
+def adjudicator_awards(request):
+    FSJ_user = get_FSJ_user(request.user.username)
+    awards_list = Award.objects.filter(committees__adjudicators = FSJ_user)
+    filtered_list = AwardFilter(request.GET, queryset=awards_list)
+    template = loader.get_template("FSJ/adj_awards_list.html")
+    context = get_standard_context(FSJ_user)
+    context["awards_list"] = awards_list
+    context["filter"] = filtered_list
+    return HttpResponse(template.render(context,request))
+
+
+@login_required
+@user_passes_test(is_adjudicator)
+def adjudicator_application_list(request, award_idnum):
+    FSJ_user = get_FSJ_user(request.user.username)
+    
+    try:
+        award = Award.objects.get(awardid = award_idnum)
+    except Award.DoesNotExist:
+        raise Http404("Award does not exist")
+
+    application_list = award.applications.all()
+
+    context = get_standard_context(FSJ_user)
+    context["application_list"] = application_list
+    context["award"] = award
+    context["is_adj"] = True
+
+    template = loader.get_template("FSJ/application_list.html")
```

Comment:
application list loads with "return to awards", "archive application(s)", and "delete application(s)" which throw a 403 forbidden as well as a "view archive" which throws a 404 page not found error.

Reply:
my mistake, did not test sufficiently after latest merge with master. will fix